docs/provided.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 3
+title: Provided Codemods 
+---
+
+You may see the term "Provided Codemod" in our documentation and in our code. This page discusses what differentiates "provided" codemods from more typical codemods.
```

Comment:
Unsolicited opinion but I don't like this term. "External Codemod" sounds better to me.